### ISSUE TITLE
Possibility to disable "Share on Twitter" feature

### ISSUE THREAD

**Issue by vprivat-ads** [radiantearth/stac-browser#510]
Twitter is no longer the friendly public agora we knew a few years ago, and I couldn't find an option to disable this integration. Is it possible add a new option to disable this?

![image](https://github.com/user-attachments/assets/45f22284-d316-4f4b-9435-d0ba061254fe)

**Comment by m-mohr** [comment]
Not right now unless you change it in the code directly. PRs welcome.

**Comment by vprivat-ads** [comment]
ok :) no objection to add other share links (twitter, bluesky, linkedin, etc.), and let the user configure which ones are enabled?

**Comment by m-mohr** [comment]
No, actually there are libraries for Vue that do that. The ones I have right now just are there because they just work out of the box without dependencies and additional code. So was just lazy me. I don't want to maintain those API specifics etc.

**Comment by m-mohr** [comment]
I found that there are also easy ways to share on Bluesky and Mastodon, so we could extend this feature very easily to those.
For details see https://github.com/Open-EO/openeo-web-editor/commit/e76574e27ee6bf46006c8171e987987a6d04c72f
That makes it less Twitter centric...

**Comment by m-mohr** [comment]
852af745 implements the following changes:
- Adds Mastodon.social
- Adds Bluesky
- Rebrands Twitter to X
- Add config option `disableSocialSharing` to support disabling the social sharing buttons (excluding e-mail)
- (SVG-related changes for the logos/icons)

Additional services via a Vue plugin are still welcome as PR. Closing for now, but feel welcome to open a PR as discussed above.

**Comment by m-mohr** [comment]
Interesting source, by the way: https://github.com/bradvin/social-share-urls

**Comment by vprivat-ads** [comment]
Thanks a lot!
